examples/programmatic/tensor_basis/tensor_basis.py

```diff
@@ -0,0 +1,191 @@
+"""
+Training a polarizability model using the TensorBasis in metatrain
+==================================================================
+
+This tutorial demonstrates how to train equivariant models in metatrain constructed as
+linear combinations of the elements of a tensorial basis with neural networks
+(SOAP-based Behler-Parrinello nets) as coefficients of the linear combination.
+"""
```

Comment:
I would avoid having the `TensorBasis` in the title. `metatrain` is mainly for people who don't know the details and just want to train. I would rather go for something more generic like "Training a polarizability model". Then, in the text, you can explain that we're using SOAP-BPNN with the `TensorBasis`